pkg/arvo/sys/hoon.hoon

```diff
@@ -262,6 +262,9 @@
 ++  tail  |*(^ ,:+<+)                                   ::  get tail
 ++  test  |=(^ =(+<- +<+))                              ::  equality
 ::
+++  lead  |*(h=* |*(* [h +<]))                          ::  put head
```

Comment:
Why |*?

Reply:
You'll want the types of whatever nouns you pass in to be preserved, so that the eventual result can nest correctly, right?

Reply:
GrrrrRRRr. Fine.